resources/app.yaml

```diff
@@ -38,6 +38,7 @@ hooks:
             containers:
               - name: hook
                 image: monitoring-hook:0.0.1
+                command: ["dumb-init", "/rollback.sh"]
```

Comment:
nit: just curious - isn't it safer to use absolute paths to avoid issues with PATH in the container?